rviz_default_plugins/src/rviz_default_plugins/displays/laser_scan/laser_scan_display.cpp

```diff
@@ -72,6 +85,7 @@ void LaserScanDisplay::processMessage(sensor_msgs::msg::LaserScan::ConstSharedPt
   if (tolerance > filter_tolerance_) {
     filter_tolerance_ = tolerance;
     tf_filter_->setTolerance(filter_tolerance_);
+    checkTolerance(RCL_NS_TO_S(filter_tolerance_.nanoseconds()));
```

Comment:
Sorry, I was a bit unclear.  I think the parameter to `checkTolerance` should be an `rclcpp::Duration`, and we can do the `RCL_NS_TO_S` conversion in there.  That should also probably be stored into `int32_t`, just so we are clear on the type.

Reply:
https://github.com/ros2/rviz/pull/1035/commits/220f86697b523cb86868f3e0e19ad2f1352b846f